recipe/meta.yaml

```diff
@@ -12,21 +12,23 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   skip: True  # [py3k or win]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.8.*  # [not py36]
+    - numpy 1.11.*  # [py36]
```

Comment:
Ok. But that doesn't explain why the build suddenly complains about a missing dependency.